cpp/include/cudf/detail/utilities/vector_factories.hpp

```diff
@@ -380,7 +382,7 @@ thrust::host_vector<T> make_host_vector_async(device_span<T const> v, rmm::cuda_
  * @brief Asynchronously construct a `std::vector` containing a copy of data from a device
  * container
  *
- * @note This function synchronizes `stream`.
+ * @note This function does not synchronize `stream`.
```

Comment:
fixed typo

Reply:
Out of the scope of the current PR, the `_sync` suffix in `make_host_vector_sync` is wordy and confusing and not aligned with CUDA/C++ naming convention: APIs are treated as synchronous by default and only asynchronous ones are named by adding a `_async` suffix. I assume this is also why the typo was introduced in the first place.